src/popup/popup.js

```diff
@@ -12,7 +12,7 @@ function updateCurrentLang() {
 
 document.querySelectorAll('a[data-lang]').forEach(function(element) {
   element.addEventListener('click', function(event) {
-    chrome.storage.sync.set({'language': event.target.getAttribute('data-lang')}, function() {
+    chrome.storage.sync.set({'language': event.target.getAttribute('data-lang'), 'displaylang': event.target.getAttribute('data-display-lang')}, function() {
```

Comment:
I don't think it makes sense for us to keep track of both values. They're totally related, and knowing one should be enough for us to know both.